subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -72,6 +72,15 @@ To ignore directories in Gradle 8.0 and later, the input property needs to be an
 Finally, using `@link:{javadocPath}/org/gradle/api/tasks/InputDirectory.html[InputDirectory]` implies `@IgnoreEmptyDirectories`, so no changes are necessary when using this annotation.
 The same is true for `@link:{javadocPath}/org/gradle/api/tasks/TaskInputs.html#dir-java.lang.Object-[inputs.dir()]` when registering an input directory via the runtime API.
 
+[[tar_tree_no_backing_file]]
+==== Tar trees from resources without backing files
+
+It is possible to create tar trees from arbitrary resources.
```

Comment:
This is just my pet peeve, but having the word "tar" without any kind of emphasis in the text is weird for me. I'd expect it to be spelled all-caps, or at least in italics. WDYT?

Reply:
all-caps it is.